tests/components/unifiprotect/test_config_flow.py

```diff
@@ -365,10 +365,7 @@ async def test_discovered_by_unifi_discovery_direct_connect_updated(
     )
     mock_config.add_to_hass(hass)
 
-    with _patch_discovery(), patch(
-        "homeassistant.components.unifiprotect.async_setup_entry",
```

Comment:
If there's a reload listener we should still patch the integration setup function and block till done so that all tasks are finished and protected before we exit the test.

Reply:
Since we aren't actually setting up the integration here the reload listener isn't going to fire in this case.

We do test that it works here https://github.com/bdraco/home-assistant/blob/97d27918c2c81b5332f10e0d1d54b8078d5d3679/tests/components/unifiprotect/test_init.py#L76

Reply:
Ok, good!